File: Day 2/JavaScript/Level 3/Task6.js
const prompt = require('prompt-sync')();

let number = Number(prompt("Enter number: "))

let sumN = 0;
for (index = number; index > 0; index--){
    sumN += index;
    }

console.log("Sum of all numbers fro 1 to", number, " =", sumN)

File: Pre-Ileya/Day 2/Java/Level 3/Task6.java
import java.util.Scanner;

public class Task6{
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter number: ");
        int number = scanner.nextInt();

        int sumN = 0;
        for (int index = number; index > 0; index--){
            sumN += index;
        }
        System.out.printf("Sum of all numbers from 1 to %d = %d%n", number, sumN);
    }
}
